api/openapi-spec/swagger.json

```diff
@@ -46070,35 +46070,6 @@
       "required": true
      }
     ]
-   },
-   "/version/": {
```

Comment:
Removing an endpoint from OpenAPI spec is a backward incompatible change. Is it possible that an OpenAPI client rely on this?